src/assets/scripts/game/GameController.js

```diff
@@ -1,13 +1,35 @@
 /* eslint-disable camelcase, no-underscore-dangle, no-mixed-operators, func-names, object-shorthand,
 no-undef, class-methods-use-this */
 import $ from 'jquery';
+import _forEach from 'lodash/forEach';
 import GameOptions from './GameOptions';
 import { round } from '../math/core';
 import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
 
 // Temporary const declaration here to attach to the window AND use as internal property
 const game = {};
 
+// TODO: Remember to move me to wherever the constants end up being moved to
+/**
+ * Definitions of point values for given score events
+ * @type {Object}
+ */
+const GAME_EVENTS = {
```

Comment:
Yeah, that is a problem.  The solution here is actually create two objects: One with a key and a value `ARRIVAL: 10` and one that is simply an enumeration: `ARRIVAL: 'ARRIVAL'`.  That may seem a little goofy, but this allows for slightly increased performance and consolidation of the event names.  Just like you've done with the scores.  You could call them `GAME_EVENT_NAMES` for the enum and `GAME_EVENT_SCORE_MODIFIERS`, or something along those lines.  Then, in the functions that call `record_game_event` you simply reference the enum: 
```
window.gameController.record_game_event('DEPARTURE');
// becomes
window.gameController.record_game_event(GAME_EVENT_NAMES.DEPARTURE);
```

That will help eliminate the possibility of typos, it references the same object and keeps everything in one place, and there is a single source of truth for what each event means.